File: tests/C/0019-present.c
int main (void)
{
	unsigned v[10] = { 0 };

	#pragma acc data create(v)
	{
		#pragma acc parallel loop present(v)
		for (unsigned i = 0; i < 10; ++i)
			v[i] = i;
		#pragma acc update host(v)
	}

	#pragma acc data create(v)
	{
		#pragma acc parallel loop default(present)
		for (unsigned i = 0; i < 10; ++i)
			v[i] = 2*i;
		#pragma acc update host(v)
	}
	return 0;
}

File: tests/Fortran/Fixed/0019-present.f
      program test
      implicit none
      integer :: v(10)
      integer :: i
      v(:) = 0

!$acc data create(v)
!$acc parallel loop present(v)
        do i = 1, 10
          v(i) = i
        end do
        write (*,*) "v(3) = ", v(3)
!$acc update host(v)
        write (*,*) "v(3) = ", v(3)
!$acc end data

!$acc data create(v)
!$acc parallel loop default(present)
        do i = 1, 10
          v(i) = i
        end do
        write (*,*) "v(3) = ", v(3)
!$acc update host(v)
        write (*,*) "v(3) = ", v(3)
!$acc end data

      end program
